addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -186,3 +186,19 @@ $code-color: $o-main-code-color!default;
 $kbd-color: $o-gray-700 !default;
 $kbd-bg: $o-gray-100 !default;
 $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($o-white, 0.8) !default;
+
+// Border-radius
+
+$border-radius-lg: $o-border-radius-lg !default;
```

Comment:
In this file we keep bootstrap's variables organisation and order ([ref.](https://github.com/twbs/bootstrap/blob/8fa0d30101/scss/_variables.scss))... this should be in the "Components" section, could you check the others too?